fe/fe-core/src/main/java/com/starrocks/catalog/AggregateType.java

```diff
@@ -97,7 +97,6 @@ public enum AggregateType {
         primitiveTypeList.clear();
         // all types except bitmap, hll, percentile and complex types.
         EnumSet<PrimitiveType> excObject = EnumSet.allOf(PrimitiveType.class);
-        excObject.remove(PrimitiveType.BITMAP);
```

Comment:
you couldn't simply remove this code.

Reply:
only remove restriction for replace, do not change others

Reply:
OK, fixed